micro-rdk-ffi/src/ffi/runtime.rs

```diff
@@ -43,6 +58,8 @@ extern "C" {
 #[no_mangle]
 pub extern "C" fn init_viam_server_context() -> *mut viam_server_context {
     let registry = Box::<ComponentRegistry>::default();
+    #[cfg(target_os = "espidf")]
+    initialize_logger::<micro_rdk::esp32::esp_idf_svc::log::EspLogger>();
```

Comment:
What prompted moving this here?

Reply:
we missed uploading some logs because we were previously initializing the logger too late